1 Table Customer.sql

```diff
@@ -0,0 +1,31 @@
+use CostumeBusinessDB
+go 
+drop table if exists dbo.Customer
+go 
+create table dbo.Customer(
+    CustomerID int not null identity primary key,
```

Comment:
Syntax tip: column names keep camel case ie: CustomerId

Reply:
Thanks, I corrected this to CustomerId.